core/lib/include/debug.h

```diff
@@ -59,6 +59,27 @@ extern "C" {
 #define DEBUG_PRINT(...) printf(__VA_ARGS__)
 #endif
 
+/**
+ * @def DEBUG_BREAKPOINT
+ *
+ * @brief Set a debug breakpoint
+ *
+ * When `DEVELHELP` is enabled, this traps the CPU and allows to debug the
+ * program with e.g. `gdb`.
+ * Without `DEVELHELP` this turns into a no-op.
+ *
+ * @warning     If no Debugger is attached, the CPU might get stuck here
+ *              and consume a lot of power until reset.
+ *
+ * @param val   Breakpoint context for debugger, usually ignored.
+ */
+#ifdef DEVELHELP
+#include "architecture.h"
+#define DEBUG_BREAKPOINT(val)   ARCHITECTURE_BREAKPOINT(val)
+#else
+#define DEBUG_BREAKPOINT(val)
+#endif
```

Comment:
```suggestion
#ifdef DEVELHELP
#include "architecture.h"
#define DEBUG_BREAKPOINT(val)   ARCHITECTURE_BREAKPOINT(val)
#else
#define DEBUG_BREAKPOINT(val)   (void)0
#endif
```

So that the `;` after `ARCHITECTURE_BREAKPOINT()` is already required?